drivers/display/display_st7789v.c

```diff
@@ -88,6 +92,14 @@ static void st7789v_transmit(struct st7789v_data *data, u8_t cmd,
 	}
 }
 
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
+static void st7789v_enter_sleep(struct st7789v_data *data)
```

Comment:
Can you move `st7789v_enter_sleep()` just above `st7789v_pm_control() ` and so reduce the number of "`#ifdef CONFIG_DEVICE_POWER_MANAGEMENT`", please ?

Reply:
Yes, fixed.